mizar/obj/networkpolicy.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020 The Authors.
+
+# Authors: Hong Chang   <@Hong-Chang>
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:The above copyright
+# notice and this permission notice shall be included in all copies or
+# substantial portions of the Software.THE SOFTWARE IS PROVIDED "AS IS",
+# WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+# TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+# FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR
+# THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import logging
+import ipaddress
+from mizar.common.common import *
+
+logger = logging.getLogger()
+
+
+class NetworkPolicy:
+    def __init__(self, name, obj_api, opr_store, spec=None):
+        self.name = name
+        self.obj_api = obj_api
+        self.store = opr_store
+        self.endpoints = []
+        if spec is not None:
+            self.set_obj_spec(spec)
+
+    @property
+    def get_endpoints(self):
+        return self.endpoints
+
+    def get_obj_spec(self):
+        self.obj = {
+            "endpoints": self.endpoints
+        }
+
+        return self.obj
+
+    def set_obj_spec(self, spec):
+        self.endpoints = get_spec_val('endpoints', spec)
+
+    def get_name(self):
+        return self.name
+
+    def get_plural(self):
+        return "networkpolicies"
+
+    def get_kind(self):
+        return "NetworkPolicy"    
+
+    def set_endpoints(self, endpoints):
+        self.endpoints = endpoints
+
+    def store_update_obj(self):
+        if self.store is None:
+            return
+        self.store.update_networkpolicy(self)
+
+    def store_delete_obj(self):
+        if self.store is None:
+            return
+        self.store.delete_networkpolicy(self.name)
+        
```

Comment:
same as above "no newline" mark needs to be removed

Reply:
fixed.